skll/utilities/compute_eval_from_predictions.py

```diff
@@ -78,6 +159,14 @@ def main(argv=None):
     parser.add_argument('metric_names',
                         help='metrics to compute',
                         nargs='+')
+    parser.add_argument('--probability', '-p',
+                        help="Whether the predictions file contains probabilities.",
+                        action='store_true')
+    parser.add_argument('--prediction_method', '-P',
+                        help="How to generate a prediction from the class probabilities "
+                             "(only use with -p flag). Supported methods: 'choose_best' "
```

Comment:
should we have an explicit check here that `--prediction_method` is not passed in without the `--probability` flag?

Reply:
Yes, we should add that.

Reply:
I will remove the `probability` flag, and generate a warning if there is a prediction method provided but the file does not contain probabilities.

Reply:
And, in the other case - where we have probabilities but the user hasn't explicitly specified a method, we should print out a logging message saying we used the default prediction method.